src/sentry/eventstream/base.py

```diff
@@ -201,7 +195,7 @@ def requires_post_process_forwarder(self) -> bool:
         return False
 
     @staticmethod
-    def _get_event_type(event: Event | GroupEvent) -> EventStreamEventType:
+    def _get_event_type(event: Event | GroupEvent) -> str:
```

Comment:
is it possible to leave these and restore the original type?

Reply:
I think this is related to enum values requiring pickle to be put into a celery task. We could use `eventstream_type.value` when scheduling the task though.